manifests/f/Famatech/AdvancedIPScanner/2.5.3850/Famatech.AdvancedIPScanner.installer.yaml

```diff
@@ -2,10 +2,20 @@
 
 PackageIdentifier: Famatech.AdvancedIPScanner
 PackageVersion: 2.5.3850
+Platform:
+- Windows.Desktop
+InstallerType: exe
```

Comment:
This is redundant when the the installer says its inno

Reply:
Removed lines 5-7.

Reply:
Oops didnt mean to mark Platform, did it a bit quick

Reply:
no worries added the platform back.